docs/sql/statements/update.rst

```diff
@@ -28,6 +29,12 @@ UPDATE changes the values of the specified columns in all rows that satisfy the
 condition. Only the columns to be modified need be mentioned in the SET clause;
 columns not explicitly modified retain their previous values.
 
+The optional RETURNING clause causes UPDATE to compute and return value(s) based
+on each row actually updated. Any expression using the table's columns, and/or
+columns of other tables mentioned in FROM, can be computed. The new (post-update)
+values of the table's columns are used. The syntax of the RETURNING list is
+identical to that of the output list of SELECT.
```

Comment:
`identical to that of the output list of SELECT` it is not really true, coz a select item in the `SELECT` can be also `schema_name.*`.

`output_expression` and `output_name` probably should be added to the `Parameters` section.

Reply:
Good point, thank you!